packages/hardhat-core/test/internal/hardhat-network/provider/modules/evm.ts

```diff
@@ -614,13 +614,15 @@ describe("Evm module", function () {
               toFake: ["Date"],
             });
 
+            const firstBlock = await mineEmptyBlock();
             await this.provider.send("evm_increaseTime", [100]);
             const snapshotBlock = await mineEmptyBlock();
+
             const snapshotId = await this.provider.send("evm_snapshot");
 
             assert.equal(
               quantityToNumber(snapshotBlock.timestamp),
-              getCurrentTimestamp() + 100
+              quantityToNumber(firstBlock.timestamp) + 100
```

Comment:
I honestly don't know why this test was working before. Now it was failing for a difference of a few milliseconds. I guess this is related to the timestamp change in #979, but I don't see how.

Anyway, now I just mine a block at the beginning of the test to compare the result of `evm_increaseTime` against that.

Reply:
Milliseconds? I think the timestamp is in seconds.